contrib/completion/bash/docker-compose

```diff
@@ -131,25 +131,25 @@ _docker-compose_kill() {
 			COMPREPLY=( $( compgen -W "--help -s" -- "$cur" ) )
 			;;
 		*)
-			__docker-compose_services_running
+			__docker_compose_services_running
 			;;
 	esac
 }
 
 
-_docker-compose_logs() {
+_docker_compose_logs() {
 	case "$cur" in
 		-*)
 			COMPREPLY=( $( compgen -W "--help --no-color" -- "$cur" ) )
 			;;
 		*)
-			__docker-compose_services_all
+			__docker_compose_services_all
 			;;
 	esac
 }
 
 
-_docker-compose_migrate-to-labels() {
+_docker_compose_migrate_to_labels() {
```

Comment:
Gah! Good eye. Without tests for this stuff (I wouldn't even know how), that's easy to miss. I'll fix it and push again tomorrow. Yay!